app/controllers/server_load_readings_controller.rb

```diff
@@ -0,0 +1,19 @@
+class ServerLoadReadingsController < ServerBaseController
+  def index
+  end
+
+  def data
```

Comment:
Show isn't right here either since it is loading all readings. But you're right this isn't nice either changed it a bit around I think this is better.